pyaptamer/aptanet/_pipeline.py

```diff
@@ -72,6 +72,15 @@ def fit(self, X, y):
         self.pipeline_ = self._build_pipeline()
         self.pipeline_.fit(X, y)
 
-    def predict(self, X):
+    def predict(self, X, output_type: str = "class"):
```

Comment:
Exact same feedback for the `predict` method in `AptaNetClassifier` (the 2 above comments)